.github/ISSUE_TEMPLATE/ feature_request copy.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Backlog item
+about: Template used internally for new features
+title: "[Backlog] "
+labels: "backlog"
```

Comment:
is it possible to have the template automatically assigning the project, and maybe have the status as backlog instead of a label?

Reply:
I searched around as this would solve one of our biggest issues with Projects, but it seems that this is not possible at the moment. I did find this action though. Perhaps it is something worth exploring.